uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/controllers/sub-controllers/request-modal-details-toil.controller.spec.js

```diff
@@ -105,7 +97,7 @@ define([
       });
 
       it('caches TOIL accrual options', function () {
-        expect(OptionGroup.valuesOf.calls.mostRecent().args[2]).not.toBe(false);
+        expect(OptionGroup.valuesOf.calls.mostRecent().args[1]).not.toBe(false);
```

Comment:
Can we name `OptionGroup.valuesOf.calls.mostRecent().args[1]` just as https://github.com/civicrm/civihr/pull/2558/files#diff-0389c1a7f834f6deb08e1dd520f7183dR346 was named?

Reply:
Done.